src/libsync/owncloudpropagator.h

```diff
@@ -227,7 +227,7 @@ class OWNCLOUDSYNC_EXPORT PropagateDirectory : public PropagatorJob {
 private slots:
     bool possiblyRunNextJob(PropagatorJob *next) {
         if (next->_state == NotYetStarted) {
-            connect(next, SIGNAL(finished(SyncFileItem::Status)), this, SLOT(slotSubJobFinished(SyncFileItem::Status)), Qt::QueuedConnection);
+            connect(next, SIGNAL(finished(SyncFileItem::Status)), this, SLOT(slotSubJobFinished(SyncFileItem::Status)));
```

Comment:
Yes, notably the invokeMethod additions in "Create PropagateItemJobs only before starting them" and all of "Don't abort propagation job abortions synchronously from finished". That's why I've put them in the same pull request.